nlu_hyperopt/app.py

```diff
@@ -4,7 +4,16 @@
 import logging
 import sys
 
-from nlu_hyperopt.space import search_space
+input_search_space = os.environ.get("INPUT_SEARCH_SPACE")
+if input_search_space:
+    import importlib.util
+    spec = importlib.util.spec_from_file_location("nlu_hyperopt.space", input_search_space)
+    space = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(space)
+    search_space = space.search_space
```

Comment:
I'd put that in a separate function, but looks good 👍

Reply:
let's also add comment why we are doing this here (e.g. that this only is used for the github actions bla)